core/thread/fetch_spec.rb

```diff
@@ -0,0 +1,15 @@
+ruby_version_is '2.5' do
+  describe 'Thread#fetch' do
+    it "returns the value of the fibre-local varible if value has been assigned" do
+      th = Thread.new { Thread.current[:cat] = 'meow' }
+      th.join
+      th.fetch(:cat, true).should == 'meow'
+    end
+
+    it "returs the default value if fibre-local variable hasn't been assigned" do
+      th = Thread.new {}
+      th.join
+      th.fetch(:cat, true).should == true
+    end
+  end
```

Comment:
Can you add specs for cases where only 1 argument is passed to Thread#fetch?

Reply:
No, I think KeyError is intended and consistent with `Hash#fetch`.

Reply:
okay. I have updated the PR.